exercise12_5.py

```diff
@@ -6,24 +6,23 @@
 Python for Everybody: Exploring Data Using Python 3
 by Charles R. Severance
 
-Solution by Jamison Lahman, June 5, 2017
+Solution by Daniel Lee, Dec 29, 2019
 """
 import socket
-import re
 
 my_sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
 my_sock.connect(('data.pr4e.org', 80))
-cmd = 'GET http://data.pr4e.org/romeo.txt HTTP/1.0\n\n'.encode()
+cmd = 'GET http://data.pr4e.org/romeo.txt HTTP/1.0\r\n\r\n'.encode()
 my_sock.send(cmd)
 
 data = my_sock.recv(512)
 message = data.decode()
 header_end_pos = message.find('\r\n\r\n') + 4   # Finds the end of header
                                             # Adds four to exclude:'\r\n\r\n'
-print(message[header_end_pos:])
+print(message[header_end_pos:],end='')
```

Comment:
Unless this is necessary, would prefer not to include. I don't think this was ever covered in the book and would like to use as little code from outside of the text as possible so it is easier to follow.